scripts/pre_commit_linter.py

```diff
@@ -1495,11 +1604,10 @@ def _match_line_breaks_in_controller_dependencies(all_files):
                 if stringfied_dependencies != function_parameters:
                     failed = True
                     print (
-                        'Please ensure that in file %s the line breaks pattern '
-                        'between the dependencies mentioned as strings:\n[%s]\n'
-                        'and the dependencies mentioned as function parameters:'
-                        '\n(%s)\nfor the corresponding controller should '
-                        'exactly match.' % (
+                        '%s --> Please ensure that line breaks between '
```

Comment:
Why this change? Please take care while merging your local branch with develop.

Reply:
Sorry for this. Corrected it.